rules-tests/Php85/Rector/FuncCall/OrdSingleByteRector/Fixture/multibyte_str.php.inc

```diff
@@ -0,0 +1,11 @@
+<?php 
+namespace Rector\Tests\Php85\Rector\FuncCall\OrdSingleByteRector\Fixture;
+$a = 'abc';
+echo ord($a);
```

Comment:
please add test for direct string value:

```
echo ord('abc');
```

should become:

```diff
-echo ord('abc');
+echo ord('a');
```

Reply:
Ok